### ISSUE TITLE
can not see downloaded Mapps

### ISSUE THREAD

**Issue by toraschi** [fransschreuder/libosmscout#55]
![screenshot20191107_190524529](https://user-images.githubusercontent.com/32172610/68993627-79c1d580-087a-11ea-922b-a269e3f27fc6.png)
![screenshot20191107_190724135](https://user-images.githubusercontent.com/32172610/68993628-79c1d580-087a-11ea-8007-c8b105e4e90e.png)

Hello I'm using OSM scout with the german Maps of Bayern and Württemberg. As there are no other Maps in the repository online available I copied the Maps of Austria and Italy, whitch I downloaded some month ago on an other device to the phablet/.local/share/osmscout.zubozrout/Maps/ 
There are the other Mapsfoder too. But OSMScout doesn't regognize them. How can I make the selectable in OSM Scout app?

**Comment by a-i-m-a-r** [comment]
Hi toraschi,
Notice that you have 22 files in some folders and 20 in others. I bet it is that Austria and Italy lack standatd.oss file.  I think it is a generica file that does not contain info for a particular country. Once happened to me. Double check it.
Notice also that you are using Martin Kozub's fork, and this git is Frans'. Anyway both are currently unavailable on openstore.

**Comment by fransschreuder** [comment]
Yes, I need to regenerate the maps, but have a lack of time to do so.

**Comment by a-i-m-a-r** [comment]
Hi Frans,
Is it necessary?
Can't toraschi copy the missing files from the other folders (supposing that I am correct and e.g. standard.oss contains only generic info)?
BTW, may I ask how do you generate the maps?
 http://data.modrana.org/osm_scout_server/osmscout-22/osmscout/ contains similar files, but are not exactly the same and do nor work in the app.
